scripts/publish_zap_compatible.py

```diff
@@ -34,7 +34,8 @@ def scan_and_remove(obj, mappings):
             specification,
             (
                 ("format", "date"),
-                ("personalisation", None)
+                ("personalisation", None),
+                ("/callbacks", None)
```

Comment:
Is this to stop us scanning the /callbacks endpoints we have defined in our spec?

Reply:
Lol just scrolled down and saw you've added a description of the changes on the PR, nice work